.kokoro/nightly/integration-beam.cfg

```diff
@@ -0,0 +1,38 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```

Comment:
@kolea2 what else needs to be setup to run these ITs from this branch?

Reply:
This is sufficient - it looks like the presubmit ran (though failed due to test errors)